Source/DataSources/ModelVisualizer.js

```diff
@@ -137,7 +137,6 @@ define([
                     url : resource.url,
                     animationsRunning : false,
                     nodeTransformationsScratch : {},
-                    originalNodeMatrixHash : {},
```

Comment:
The original node matrix is now maintained by the node itself, not a private hash here in the visualizer.  This was needed because articulations and nodeTransformations both need access to the original matrix per node, and would step on each other's hashing of the matrix.